R/learner_glmnet_surv_cv_glmnet.R

```diff
@@ -108,16 +123,31 @@ delayedAssign(
           pv$weights = task$weights$weight
         }
 
-        glmnet_invoke(data, target, pv, cv = TRUE)
+        list(
+          model = glmnet_invoke(data, target, pv, cv = TRUE),
+          # need these for distr prediction
+          x = data,
```

Comment:
It needs to be possible to disable this, we don't always want to store the complete training data.
This changes the size of the stored model by a pretty big factor I think when we have large datasets and should be something to be considered. 

I suggest the following:
there should be a parameter that turns the saving of the model (and hence whether a `distr` prediction can be made) on and off. This parameter should be `FALSE` per default. I think we should really strive to avoid very large objects in the `$model` slot.

Reply:
we then also need to throw a proper error when the `distr` predict type is requested during `$.predict` but the required information is not available.